hbase-server/src/main/java/org/apache/hadoop/hbase/master/HMasterCommandLine.java

```diff
@@ -62,7 +62,9 @@
     "   --localRegionServers=<servers> " +
       "RegionServers to start in master process when in standalone mode.\n" +
     "   --masters=<servers>            Masters to start in this process.\n" +
-    "   --backup                       Master should start in backup mode";
+    "   --backup                       Master should start in backup mode" +
+    "   --forceStop                    " +
```

Comment:
I think `forceStop` is generally referred to when we want to non-gracefully shutdown any service. Let's use flag `--shutDownCluster` instead.